contracts/evoting/types/election.go

```diff
@@ -409,3 +416,109 @@ type PubsharesUnits struct {
 	// PubsharesUnit
 	Indexes []int
 }
+
+// AddVoter add a new voter to the form.
+func (form *Form) AddVoter(userID string) error {
+	sciperInt, err := strconv.Atoi(userID)
+	if err != nil {
+		return xerrors.Errorf("Failed to convert SCIPER to an INT: %v", err)
+	}
+
+	form.Voters = append(form.Voters, sciperInt)
+
+	return nil
+}
+
+// GetVoterIndex return the index of voter if userID is one, else return -1
+func (form *Form) GetVoterIndex(userID string) (int, error) {
+	sciperInt, err := SciperToInt(userID)
+	if err != nil {
+		return -1, xerrors.Errorf("Failed SciperToInt: %v", err)
+	}
+
+	for i := 0; i < len(form.Voters); i++ {
+		if form.Voters[i] == sciperInt {
+			return i, nil
+		}
+	}
+
+	return -1, nil
+}
+
+// RemoveVoter remove a voter to the form.
+func (form *Form) RemoveVoter(userID string) error {
+	index, err := form.GetVoterIndex(userID)
+	if err != nil {
+		return xerrors.Errorf("Failed GetVoterIndex: %v", err)
+	}
+
+	if index < 0 {
+		return xerrors.Errorf("Error while retrieving the index of the element.")
+	}
+
+	form.Voters = append(form.Voters[:index], form.Voters[index+1:]...)
+	return nil
+}
+
+// AddOwner add a new owner to the form.
+func (form *Form) AddOwner(userID string) error {
+	sciperInt, err := SciperToInt(userID)
+	if err != nil {
+		return xerrors.Errorf("Failed SciperToInt: %v", err)
+	}
```

Comment:
I would use a more generic error message "failed to add owner: %v" or so, since "SciperToInt" is not really meaningful unless you know the code very well

Reply:
I see your point that SciperToInt is not really meaningful, but as the error are built,
if the Sciper to int fails (called from AddOwner)
it will return Failed SciperToInt
which will then be added to the failure of evoting.go 879
so the final error will be ...: couldn't add owner: Failed SciperToInt: SCIPER %v is out of range.
with the sciper number for instance.
In my opinion it is useful to build a stack trace, but if you think it is useless I can remove it 😅.

Reply:
As discussed on slack, I've changed the "Failed SciperToInt" to "failed to convert SCIPER to integer" to improve code clarity.